contrib/cmd/k8s-broker/k8s-broker.go

```diff
@@ -55,12 +55,14 @@ func main() {
 	}
 
 	if len(options.HelmBinary) == 0 {
-		panic("Need helm_binary specified")
+		fmt.Fprintln(os.Stderr, "--helm_binary must be specified")
+		os.Exit(1)
```

Comment:
@duglin in general, I agree, but the cli-ish nature of this is incidental here. Keep in mind how we expect this binary to actually be used-- in a container in a pod in a k8s cluster...

If this error occurred in such a context, you'd want to see the error timestamped, aggregated, and indexed according to the same rules as all other log messages from the same container.

So now that I think about this further, I would argue this should actually use `glog`.